AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -211,7 +212,9 @@
     <ClInclude Include="Stdafx.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="AssemblyInfo.cpp" />
+    <ClCompile Include="AssemblyInfo.cpp">
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
That's probably the default; you might have clicked on it in the UI to cause it to become an explicit override here. So, just delete this element.

---
In reply to: [75560858](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75560858) [](ancestors = 75560858,75553518,75537238,75174478)